thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/anomaly/onboard/DetectionOnboardServiceExecutorTest.java

```diff
@@ -30,7 +30,7 @@ public void shutdownExecutor() {
   @Test
   public void testCreate() throws InterruptedException {
     long jobId = executor.createDetectionOnboardingJob(
-        new DummyDetectionOnboardJob("NormalOnboardJob", Collections.<String, String>emptyMap()));
+        new DummyDetectionOnboardJob("NormalOnboardJob", OnboardingTaskTestUtils.getJobProperties()));
```

Comment:
This is different type of job and it shouldn't take the properties for onboarding job. Please use the original empty map.